tests/operators/test_python.py

```diff
@@ -26,6 +26,7 @@
 from typing import List
 
 import pytest
+from parameterized import parameterized
```

Comment:
Can we use this chance to convert `TestShortCircuitOperator` away from `unittest.TestCase`? `pytest.mark.parametrize` is better than `parameterized` in many aspects.

Reply:
Of course, not a problem.